DEVELOP.md

```diff
@@ -9,65 +9,66 @@ Dependencies:
 4. [RabbitMQ](http://www.rabbitmq.com/) for workers
 
 
-### Setting up CycleDash
+### 1. Install Dependencies
+
+```bash
```

Comment:
I'd note that is for a Mac install, and that other OSs would have similar procedures (using e.g. apt-get or yum).